### DISCUSSION TITLE
Can I import users those logged in with only social accounts?

### DISCUSSION THREAD

**Discussion by meotimdihia** [ory/kratos#1317] [Q&A] [answered]
I read the issue: https://github.com/ory/kratos/issues/605

But my website has used Next-Auth https://github.com/nextauthjs/next-auth with only passwordless and log in with Google, Facebook account. 
In my case, I don't know if I can manually add their accounts in Kratos v6, as I currently don't store the password in the database. 
And I want to migrate to Kratos, even if the user must be forced to create a password in Kratos.

**Comment by Benehiko** [comment, marked as answer]
Hi @meotimdihia 

Thank you for reaching out :)

Yes you can import the identities, but they will need to create a password using an email recovery link.
https://www.ory.sh/kratos/docs/reference/api#create-an-identity

**Reply by meotimdihia** [reply]
@Benehiko thanks for the confirmation. I have to migrate new authentication because the old authentication only offers passwordless that doesn't work.
It causes more problems in real cases. And I feel the Ory stack will help me in the future.

**Reply by meotimdihia** [reply]
@Benehiko Now I understand the basic flow of how Kratos registration, sign up, ... And have built a frontend in next.js that work with Kratos.
But I stuck at import OIDC account for existing users. And it looks like Kratos currently can't import OIDC accounts yet?

**Edit**: I tried to manually create users in the Kratos database and it works perfectly
